### DISCUSSION TITLE
Error copying data to system clipboard in docker container

### DISCUSSION THREAD

**Discussion by ErikXu** [tconbeer/harlequin#431] [Q&A, Troubleshooting, Support] [answered]
**Describe the bug**
Can not copy data to clipboard in docker container.

**To Reproduce**
- Dockerfile

``` 
FROM python:3.11

RUN pip install harlequin-mysql
```

- Build docker image

``` bash
docker build -t harlequin-mysql:latest -f ./Dockerfile .
``` 

- Run docker container

``` bash
docker run --rm -it \
harlequin-mysql:latest \
harlequin -a mysql -h localhost -p 3306 -U root --password example --database dev

# or

docker run --rm -it \
harlequin-mysql:latest \
sh -c "harlequin -a mysql -h localhost -p 3306 -U root --password example --database dev"

# or

docker run --rm -it \
harlequin-mysql:latest \
bash -c "harlequin -a mysql -h localhost -p 3306 -U root --password example --database dev"
``` 

- Execute `Ctrl + C`

![image](https://github.com/tconbeer/harlequin/assets/6275608/bf9a6ec4-b88a-4f6e-a2fc-657577e03d0d)

**Expected behavior**
Should be able to copy and paste column data.

**Actual behavior**
A clear and concise description of what happened; please include screenshots or animated gifs of Harlequin if possible.

Please refer `To Reproduce` section.

**Additional context**
What is the output of `harlequin --version`?

``` 
harlequin, version 1.13.0

Installed Adapters:
  - mysql, version 0.1.2
  - duckdb, version 1.13.0
  - sqlite, version 1.13.0
``` 

What database adapter are you using with Harlequin? (Default is DuckDB)
MySQL

Can you tell us more about your system?
- Terminal: MobaXterm

For example, for my system, these are:
- Bash

**Comment by tconbeer** [comment, marked as answer]
I think this is expected -- you'll need an x window manager in your container to use a clipboard. See https://harlequin.sh/docs/troubleshooting/copying-and-pasting

**Comment by tconbeer** [comment]
(The docs are a little out of date; copying from the results viewer or data catalog now works the same way as copying from the query editor)

**Comment by ErikXu** [comment]
Thanks for your response. I finally solved this problem by following your guidance.

BTW, for kindly remind, I plan to search a solution in  [Troubleshooting](https://github.com/tconbeer/harlequin/blob/main/troubleshooting/duckdb-version-mismatch) page mentioned in `README.md`. But the page returns 404.

This makes me feel the troubleshooting in <https://harlequin.sh/> would return 404 either, but actually it's not a 404 page. (my fault 😂)

**Reply by tconbeer** [reply]
Thanks for the heads up on the broken link in the README. I've fixed that now.
